### ISSUE TITLE
Can't resolve 'apollo-client/util/observableToPromise'

### ISSUE THREAD

**Issue by steef17** [morrys/wora#55]
Yesterday I got the the following error.
``Module not found: Can't resolve 'apollo-client/util/observableToPromise' in '***\node_modules@wora\apollo-offline\lib'``

It turned out to be @wora/apollo-offline having references to `apollo-client/util/observableToPromise`. The problem here is that since _apollo-client@2.6.9_ the folder structure changed. Solved it for now by adding apollo-client to my own project on a fixed version. Since apollo-client@2.6.9 the folder structure changed from `apollo-client/util/observableToPromise` to `apollo-client/bin/util/observableToPromise`.

In ApolloClientOffline there is a reference to a file within apollo-client by following the old folder structure. 
```
var observables_1 = require("apollo-client/util/observables");
var observableToPromise_1 = tslib_1.__importDefault(require("apollo-client/util/observableToPromise"));
```

**Comment by morrys** [comment]
hi @steef17, I would not have expected a change of structure from version 2.6.8 to 2.6.9.

I will make checks, in the meantime thanks for the report

**Comment by morrys** [comment]
Opened an issue in Apollo client:
https://github.com/apollographql/apollo-client/issues/6272

**Comment by morrys** [comment]
the problem is related to a bug in apollo-client which will be corrected with the 2.6.10 release.

More details in the issue linked above.
